docs/developer-guide/maps-configuration.md

```diff
@@ -886,6 +887,7 @@ The `symbolizer` could be of following `kinds`:
   - `size` size of the icon
   - `opacity` opacity of the icon
   - `rotate` rotation of the icon
+  - `bringToFront` this boolean will allow setting the `disableDepthTestDistance` value for the feature. This would only apply on Cesium maps.
```

Comment:
```suggestion
  - `msBringToFront` this boolean will allow setting the `disableDepthTestDistance` value for the feature. This would only apply on Cesium maps.
```